open-metadata-implementation/governance-servers/open-lineage-services/README.md

```diff
@@ -69,7 +69,23 @@ With the following body:
 ```
 POST {{base-url}}/open-metadata/admin-services/users/{{user-id}}/servers/{{server-id}}/instance
 ```
-# Open Lineage Services
+
+#Shutting down the Open Lineage Services
```

Comment:
The Open Lineage Services provide a subsystem of an Open Lineage Server (an OMAG Server).  The Open Lineage Server is a Governance Server.   This means it has the Repository Services subsystem and the Open Lineage Services subsystem enabled.

It is possible to shut down and start up a server.   It is possible to configure and unconfigure a subsystem from a server.

If you unconfigure the open lineage services from an open lineage server and then restart the server, it will only have the audit log running and would not be any use.